server/methods.js

```diff
@@ -116,7 +135,7 @@ methods.importProductImages = function () {
       const productId = product._id;
       if (!Media.findOne({ "metadata.productId": productId })) {
         const shopId = product.shopId;
-        const filepath = "custom/images/" + productId + ".jpg";
+        const filepath = "plugins/reaction-swag-shop/images/" + productId + ".jpg";
```

Comment:
This will only work, if `reaction-cli` provides the new build step: "Asset provisioning"